packages/job-worker/src/ingest/rundownInput.ts

```diff
@@ -432,6 +437,36 @@ export async function handleUpdatedSegmentRanks(
 	)
 }
 
+function updateAdLibActionRanks(segmentExternalId: string, newRank: number, cache: CacheForIngest): void {
+	const adLibActionsForSegment: AdLibAction[] = cache.AdLibActions.findFetch((adLibAction: AdLibAction) =>
+		adLibAction.externalId.startsWith(segmentExternalId)
+	)
+
+	for (const adLibAction of adLibActionsForSegment) {
+		const oldRankFraction: number = (adLibAction.display._rank ?? 1) % 1
```

Comment:
Any reason why `?? 1` instead of `?? 0`? I assume that 0 is the intended default value.

Reply:
`1` was intentional, but mostly to ensure that he rank did have a decimal value. 
I don't think that's a strong argument so I have updated it to be `0`.